typings/index.d.ts

```diff
@@ -56,7 +56,7 @@ declare module 'discord.js' {
 			isWebhook?: boolean
 		): MessageOptions | WebhookMessageOptions;
 
-		public makeContent(): string | string[];
+		public makeContent(): string | string[] | void;
```

Comment:
Small nit: This should be `undefined` since `void` is not assignable to `undefined`.